apiserver/facades/client/usermanager/usermanager.go

```diff
@@ -348,13 +349,17 @@ func (api *UserManagerAPI) UserInfo(ctx context.Context, request params.UserInfo
 // information needed but not contained in the core user from the access
 // service.
 func (api *UserManagerAPI) infoForUser(ctx context.Context, tag names.UserTag, user coreuser.User) params.UserInfoResult {
+	var conn *time.Time
+	if !user.LastLogin.IsZero() {
+		conn = &user.LastLogin
+	}
```

Comment:
`conn` implies connection. Maybe `lastLogin` will be better.

Reply:
Fair, changed it